web/html/src/manager/notifications/notification-messages.js

```diff
@@ -153,15 +201,7 @@ class NotificationMessages extends React.Component {
   };
 
   decodeTypeText = (rawType) => {
-    var typeText;
-    switch(rawType) {
-      case 'OnboardingFailed': typeText = t('Onboarding failed'); break;
-      case 'ChannelSyncFailed': typeText = t('Channel sync failed'); break;
-      case 'ChannelSyncFinished': typeText = t('Channel sync finished'); break;
-      case 'CreateBootstrapRepoFailed': typeText = t('Creating Bootstrap Repository failed'); break;
-      case 'StateApplyFailed': typeText = t('State apply failed'); break;
-    }
-    return typeText;
+    return t(_MESSAGE_TYPE[rawType].text);
```

Comment:
The `t()` here will not work. As we use gettext, we have an automated parser which evaluate the `t()` functions and get the text out of it which should be translated. This will not work here.

Please add the `t()` to the _MESSAGE_TYPE dict directly.

Reply:
Argh, good catch. I still need to make up my mind about it. Thanks for pointing it out :+1: 
Fixed